libraries/joomla/form/form.php

```
@@ -1880,7 +1874,7 @@ protected function validateField($element, $group = null, $value = null, $input
 			// If the object could not be loaded return an error message.
 			if ($rule === false)
 			{
-				return new JException(JText::sprintf('JLIB_FORM_VALIDATE_FIELD_RULE_MISSING', $type), -2, E_ERROR);
+				return new JException(sprintf('Validation Rule missing: %s', $type), -2, E_ERROR);
```

I think we should throw an UnexpectedValueException here.